build.sbt

```diff
@@ -25,7 +25,7 @@ libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.2
 // Akka dependencies
 libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-http-core" % "10.0.11",
-  "com.typesafe.akka" %% "akka-http-spray-json" % "10.1.5"
+  "com.typesafe.akka" %% "akka-http-spray-json" % "10.1.5",
```

Comment:
Isn't this comma unnecessary and even wrong?

Reply:
Unsure here... As it compiles it should be correct, but still... strange

Reply:
You a right, it is compilable but should not be there. I removed it.